content/posts/irods-development-update-april-2023.md

```diff
@@ -0,0 +1,222 @@
+
+We've fixed some important issues in the Logical Quotas rule engine plugin. It now uses the local rodsadmin user to update quota values. This avoids potential permission issues. Not only that, but it no longer crashes the agent when given unsupported rule text via `irule`.
+
+Development on the new iRODS CLI has started again. We don't have any specifics at this time, but as soon as we have something to show, you'll know about it.
+
+The Testing Environment has received a nice improvement in regard to topology testing. Topology tests can now be run in parallel. Before this change, the tests took upwards of 8 hours to complete. With this latest change, we've witnessed the tests taking as little as 2.25 hours to complete.
+
+Initial work on the iRODS S3 API has been merged into upstream. This is still a highly experimental project, so there are still quite a few things to resolve before it's ready for prime time.
+
+Due to feedback from the community and what we learned working on the iRODS S3 API, we have started working on a full rewrite of the C++ REST API. The primary goals are to decrease the number of ports needed to run the application, consolidate all binaries, and add support for parallel transfer. We'll have more information as the rewrite is fleshed out. Stay tuned.
+
+It's been a long time coming, but we've finally released a new version of Metalnx. This release sets us up to start cleaning up the Metalnx project. Specifically, removing the local database required by Metalnx.
+
+Last but not least, Python iRODS Client 1.1.7 and Automated Ingest 0.4.1 have been released.
+
+That wraps up this month's update.
+
+See you next month!
+
+
+### April Technology Working Group
+
+- [Imaging Working Group](https://github.com/irods-contrib/irods_working_group_imaging)
+
+    - initial effort is an OMERO integration
+    - did not receive CZI funding
+    - looking at UNC and/or NSF funding opportunities
+    - working meeting
+        - Thur, April 20, 2023, 10am ET
+
+- [Metadata Templates Working Group](https://github.com/irods-contrib/irods_working_group_metadata_templates)
+
+    - working on whitepaper covering WG activities and findings
+    - working meeting
+        - Tues, April 18, 2023, 10am ET
+
+- [Authentication Working Group](https://github.com/irods-contrib/irods_working_group_authentication)
+
+    - building a new API endpoint to provide a flexible authentication mechanism
+    - new authentication plugins will drive the flow
+    - working meeting
+        - Tues, April 25, 2023, 10am ET
+
+- [S3 Working Group](https://github.com/irods-contrib/irods_working_group_s3)
+
+    - develop tools to present iRODS as S3-compatible storage to existing S3 clients
+    - planning and investigation into custom C++ S3 implementation
+    - working meeting
+        - Fri, May 5, 2023, 3pm ET
+
+- [TRiRODS](https://irods.org/trirods)
+
+    - Wed, May 10, 2023, 4pm ET
+    - TBD
```

Comment:
the roadmap?

Reply:
That wasn't included in the pc/twg notes.